src/ontology/Makefile

```diff
@@ -1002,3 +1002,21 @@ bg-clean:
 
 succeed:
 	echo make succeeded
+	
+
+###### NICO MIGRATION: overwriting unsuccessful goals!
+
+../sparql/construct-ecs-from-xrefs.sparql:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+d2p.pro:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+reports/semantic-xref-pairs.tsv:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+imports/ncbitaxon_import.owl:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped, because of out of memory errors with 16GB memory!" && touch $@
+	
+imports/hgnc_import.owl:
```

Comment:
we need a ticket for this, this is important

for now we can simply use an archive version. My copy in mirror/ is fine, I can send

Reply:
Fixing this as we speak see slack.

Reply:
https://github.com/monarch-initiative/mondo/pull/1853